public/js/office.web.js

```diff
@@ -191,78 +192,101 @@ $(() => {
   	}
   }
 
-  function enterInOffice(matrixProfile) {
 
-    var currentRoom = getUrlRoom();
-    if(currentRoom==null || currentRoom==undefined || currentRoom== "undefined"){
-    	currentRoom = getLastRoom(matrixProfile);
-    }
-    // make connection
-    const socket = io.connect(`${window.location.protocol}//${window.location.host}`, {
-      query: `user=${matrixProfile.loadStoredProfileAsString()}${currentRoom ? `&room=${currentRoom}` : ''}`,
-    });
+  function syncOffice(usersInRoom){
+    for (var key in usersInRoom) {
+        userInroom = usersInRoom[key];
+        showUserInRoom(userInroom.user, userInroom.room);
+      }
+  }
 
-    enterRoom.on('click', (e) => {
-      const room = $(e.target).attr('room-id');
-      const roomName = $(e.target).attr('room-name');
-      const disableMeeting = new Boolean($(e.target).attr('room-disable-meeting'));
-      socket.emit('enter-room', { room, user: matrixProfile.loadStoredProfile() });
+  function confirmRoomEnter(user,roomId, callback){
+    var r = confirm(user.name +" está chamado você para "+ getRoomName(roomId));
```

Comment:
As we do not have a well-defined way to handle language and strings, we should keep all strings in the same language, rewrite it to English sounds better, what do you think?

Reply:
yes... I forgot to solve this sentence...